frontend/apps/reader/modules/readergesture.lua

```diff
@@ -61,6 +61,10 @@ local action_strings = {
     show_config_menu = _("Show bottom menu"),
     show_frontlight_dialog = _("Show frontlight dialog"),
     toggle_frontlight = _("Toggle frontlight"),
+    increase_frontlight = _("Increase frontlight"),
```

Comment:
Maybe something like `Increase frontlight intensity` or `brightness`?

Reply:
+1 for `Brightness` (or `Frontlight Brightness`), because it pairs well with a potential `Warmth` counterpart ;).

Reply:
Increase frontlight brightness / Decrease frontlight brightness

Reply:
Sounds good to me!